canonical-websites/snapcraft.io
# Add multiselect component and wire it up for country black/white lists

Fixes https://github.com/canonical-websites/snapcraft.io/issues/1072
Fixes https://github.com/canonical-websites/snapcraft.io/issues/1074
Fixes https://github.com/CanonicalLtd/snapcraft-design/issues/490

UX: https://github.com/canonical-websites/snapcraft.io/issues/784
Design: (tags UI) https://github.com/CanonicalLtd/snapcraft-design/blob/master/snapcraft.io/market%20/Snap%20details%20-%20Listing%20view%20-%20Tags%20UI%20%20REV02.jpg

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/account
- Click on a snap from your list of snaps
- Scroll down to the bottom of the 'Listing' page
- Change the checkbox from 'All territories' to 'Custom' under 'This snap should be available in:'
- Select either option and click on the input
- Try filtering countries by typing
- Use the mouse to scroll up and down the options
- Click on options to add them
- Use the arrow keys to scroll up and down the items
- Use the return key to add them
- Use backspace to remove your search term
- Use backspace when the search term is empty to delete the item's you've added, one by one
- Click off the element to lose focus
- Lose focus while a search term is types
- Reclick the element, the search drop-down should reappear with the term you previously searched for
- Do anything else you can think of
- ## edit fields on the Listing page to make sure that still works

# Marked as WIP until an API error is sorted - Saving works, but doesn't get served back... You can however check https://dashboard.snapcraft.io to see the changes have taken effect.
